spec/controllers/build_allocations_controller_spec.rb

```diff
@@ -194,19 +185,13 @@ def further_info
 
         let(:pom_nomis_id) { poms.last.staffId }
 
-        it 'sends a de-allocation email with the correct info' do
-          expect(PomMailer).to have_received(:deallocation_email)
-            .with(
-              previous_pom_name: poms.last.firstName.capitalize,
-              responsibility: 'responsible',
-              previous_pom_email: poms.last.email_address,
-              new_pom_name: pom.full_name,
-              offender_name: "#{offender.fetch(:lastName)}, #{offender.fetch(:firstName)}",
-              offender_no: offender_no,
-              prison: prison.name,
-              url: Rails.application.routes.default_url_options[:host] + prison_staff_caseload_path(prison, pom.staffId),
-              further_info: further_info
-            )
+        it 'sends the correct emails' do
+          expect(EmailService).to have_received(:send_email).with(
+            message: notes,
+            allocation: anything,
+            pom_nomis_id: pom.staffId,
+            further_info: further_info,
+          )
```

Comment:
Does this not reduce specificity/functional coverage of the tests?

Reply:
See other comment.